config/default.yml

```diff
@@ -955,6 +955,29 @@ RSpec/Rails/HaveHttpStatus:
   VersionAdded: '2.12'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/HaveHttpStatus
 
+RSpec/Rails/InferredSpecType:
+  Description: Identifies redundant spec type.
+  Enabled: pending
+  Safe: false
+  VersionAdded: '2.14'
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/InferredSpecType
+  Inferrences:
```

Comment:
Please correct me if I'm mistaken, this option needs metadata to prevent from [being overriden by the user setting](https://docs.rubocop.org/rubocop/1.36/configuration.html#inheritance-of-hashes-vs-other-types):

> Other types, such as AllCops / Include (an array), are overridden by the child setting.

> However, advanced users can still merge arrays using the inherit_mode setting. See "Merging arrays using inherit_mode" below.

Please see [an example of how we do this for the language config](https://github.com/rubocop/rubocop-rspec/blob/87f3ef5e6b3df4a388901905f53f3bba7f0d3a13/config/default.yml#L10).

[Semi-related issue in `rubocop` repo](https://github.com/rubocop/rubocop/issues/9325).

I don't think there is a way to check that the defaults are still there from our specs, since we just stub `cop_config` and override, too.

Reply:
> Inferences is a Hash

I've missed that fact altogether.
Thanks for double-checking with a test project!
Please accept my apologies for bothering.